src/taipy/core/data/mongo.py

```diff
@@ -43,7 +43,7 @@ class MongoCollectionDataNode(DataNode):
         edit_in_progress (bool): True if a task computing the data node has been submitted
             and not completed yet. False otherwise.
         properties (dict[str, Any]): A dictionary of additional properties. Note that the
-            _properties_ parameter must at least contain an entry for _"db_name"_, _"collection_name"_, and _"custom_document"_.
+            _properties_ parameter must at least contain an entry for _"db_name"_ and _"collection_name"_.
```

Comment:
custom_document is still used even if it is not mandatory. Can you add a small comment saying it is optional, and is a class representing a mongo document to read/write from the mongo collection.

Reply:
Since there is other optional parameters, should we also explain those? Like the hostname, port, or the extra_args?

Reply:
The config API is imported dynamically in taipy-core, so currently there is no ref manual for that (I think). It would be better if we can include those. But for now I think this is sufficient.